library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -1340,6 +1341,43 @@ void vtkF3DRenderer::ShowEdge(bool show)
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::SetUseOrthographicProjection(bool use)
+{
+  // if the internal state is already the same as the target state there's nothing to do
+  if (this->UseOrthographicProjection == use)
+    return;
+
+  vtkCamera* camera = GetActiveCamera();
+  const double angle = vtkMath::RadiansFromDegrees(camera->GetViewAngle());
+  const double* position = camera->GetPosition();
+  const double* focal = camera->GetFocalPoint();
+
+  if (this->UseOrthographicProjection != use)
```

Comment:
```suggestion
  if (use)
```
you've already done the comparison of `this->UseOrthographicProjection` and `use` at the top to return early.
At this point you only need to check the value of `use` knowing that `use == true` (`if(use)`) means going from perspective to ortho and `use == false` (`else`) means going from ortho to perspective

Reply:
Thanks, you are a much better programmer than me :-P